openstack/loadbalancer/v2/apiversions/requests.go

```diff
@@ -0,0 +1,13 @@
+package apiversions
+
+import (
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// ListVersions lists all the load balancer API versions available to end-users.
+func ListVersions(c *gophercloud.ServiceClient) pagination.Pager {
```

Comment:
It's not in any docs or anything but it seems that it will be better if we try to use the common `List` name for this particular exported function.

We already have it in [sharedfilesystems](https://github.com/gophercloud/gophercloud/blob/master/openstack/sharedfilesystems/apiversions/requests.go#L9), [containerinfra](https://github.com/gophercloud/gophercloud/blob/master/openstack/containerinfra/apiversions/requests.go#L9) packages.